### PR TITLE
Optional calculation of bulk statistics in feature detection on raw input (before smoothing)

### PR DESCRIPTION
**Pull request by JuliaKukulies**
Addresses #440 and allows for the user to choose whether the bulk statistics in the feature detection are calculated on the unsmoothed or the smoothed input data. 

* [x] Have you followed our guidelines in CONTRIBUTING.md? 
* [x] Have you self-reviewed your code and corrected any misspellings? 
* [x] Have you written documentation that is easy to understand?
* [x] Have you written descriptive commit messages? 
* [x] Have you added NumPy docstrings for newly added functions? 
* [x] Have you formatted your code using black? 
* [x] If you have introduced a new functionality, have you added adequate unit tests?
* [x] Have all tests passed in your local clone? 
* [x] If you have introduced a new functionality, have you added an example notebook?
* [x] Have you kept your pull request small and limited so that it is easy to review? 
* [x] Have the newest changes from this branch been merged?

### CODE REVIEW

File: tobac/feature_detection.py [hunk 1]
```diff
@@ -1005,6 +1009,14 @@ def feature_detection_multithreshold_timestep(
     # get actual numpy array and make a copy so as not to change the data in the iris cube
     track_data = data_i.core_data().copy()
 
+    # keep a copy of the unsmoothed data (that can be used for calculating stats)
```
Comment (w-k-jones) [inline tobac/feature_detection.py:1012]: Do we need to copy the data here, or can we just pass `data_i` directly to `get_statistics` if calculating on the unsmoothed data?

Comment (JuliaKukulies) [reply tobac/feature_detection.py:1012]: No, you are right. I changed this to directly pass `data_i` to `get_statistics()`. Thanks!

**Comment by JuliaKukulies** [comment]
Something is going on with the formatting although I seem to use a compatible black version but  I had some issue with pre-commit. Will figure it out soon hopefully!

**Review by w-k-jones — APPROVED** [review]

**Review by freemansw1 — APPROVED** [review]
Thanks for the work on this, @JuliaKukulies . The one test failing isn't a concern.

**Comment by JuliaKukulies** [comment]
OK, so merge ? :)

**Pull request merged**